_sass/blocks/_all.scss

```diff
@@ -1,15 +1,27 @@
 @import 'header';
 @import 'footer';
 @import 'banner';
-@import 'communities_intro';
-@import 'communities_content';
-@import 'revenues/revenues_steps';
-@import 'revenues/revenues_forms';
-@import 'revenues/revenues_intro';
-@import 'revenues/revenues_footer';
-@import 'revenues/revenues_subpage_intro';
-@import 'revenues/revenues_subpage_steps';
-@import 'revenues/revenues_subpage_nav';
-@import 'revenues/revenues_subpage_involved';
+
+// Case Studies
+//
+// Styles for the page at /case-studies/
+//
+// Styleguide blocks.case-studies
+@import 'case-studies/_intro';
```

Comment:
Imported partial `case-studies/_intro` should be written as `case-studies/intro`